### DISCUSSION TITLE
Update Hasura on Heroku

### DISCUSSION THREAD

**Discussion by Fintasys** [hasura/graphql-engine#8107] [Question]
I originally setup Hasura on Heroku by on-click. Now I tried to update Hasura following the introductions [here](https://hasura.io/docs/latest/graphql/core/deployment/deployment-guides/heroku.html#updating-graphql-engine). 
```
git clone https://github.com/hasura/graphql-engine-heroku
cd graphql-engine-heroku
heroku git:remote -a my-hasura-app
heroku stack:set container -a my-hasura-app
git push heroku master
```
But it ends up with an error message
```
➜  graphql-engine-heroku git:(master) git push heroku master
To https://git.heroku.com/my-hasura-app.git
 ! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'https://git.heroku.com/my-hasura-app.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```
The branch is up to date tho.
Also one more thing I don't understand is that the git repo doesn't include any docker file etc. which could be updated 🤔 

Can anyone enlighten me? Thank you

**Comment by FedericoAliasCo** [comment]
could you upgrade hasura engine on heroku?
